### ISSUE TITLE
Interpretation of negative semantic similarity

### ISSUE THREAD

**Issue by Sonta** [UKPLab/sentence-transformers#760]
First, thanks a lot for the great work on this codebase and the pre-trained models!

I was wondering if you could help me get a conceptual understanding about negative similarity values. Mathematically, it's pretty obvious that getting a negative cosine similarity would simply mean that the vectors are pointing in opposite directions. But what would that mean semantically?

I've been trying to think of two sentences that would have a semantic similarity close to -1 but can't come up with anything. Do you happen to have any examples where you've had high negative values in real sentences?

Best,
Son

**Comment by nreimers** [comment]
As usual it depends on your training data and the type of loss your are using. Are all labels positive => Then you will not really experience negative scores. 

Do you have training pairs with negative scores => Then the model tries to learn this and put these example in opposite directions in vector space

**Comment by Sonta** [comment]
Sorry, should have added that I'm using one of your pre-trained models, the `paraphrase-distilroberta-base-v1`

I take it that I should be looking into the training data that was used for this model?

**Comment by nreimers** [comment]
That model was trained so that paraphrase sentences are as close as possible and non-paraphrase sentences are as distant as possible. 

I would not try to interpret to much into negative scores besides that they have a low similarity.

**Comment by Sonta** [comment]
Ok great, thanks for the clarification!
